### PR TITLE
[post-tex] Drop PFS support

### PR DESCRIPTION
**Pull request by vuntz**
Pull from source hasn't been maintained for quite some time and it's
very likely broken. So drop support for it.

### CODE REVIEW

**Comment by mjura** [comment]
For me it is OK +1

**Comment by tboerger** [comment]
LGTM

File: chef/cookbooks/glance/recipes/common.rb [hunk 1]
```diff
@@ -22,40 +22,9 @@
   action :install
 end
 
-unless node[:glance][:use_gitrepo]
-  package "glance" do
-    package_name "openstack-glance" if %w(redhat centos suse).include?(node.platform)
-    options "--force-yes" if %w(debian ubuntu).include?(node.platform)
```
Comment (jsuchome) [inline chef/cookbooks/glance/recipes/common.rb]: Any reason for deleting this line?

Comment (jsuchome) [inline chef/cookbooks/glance/recipes/common.rb]: Ah, sorry, I overlooked the second commit's message.

**Comment by jsuchome** [comment]
+1 (tested some glance commands after applying the patch)

**Pull request merged**